packages/react/src/components/Cta/__stories__/Cta.stories.js

```diff
@@ -12,70 +12,109 @@ storiesOf('Components|CTA', module)
       sidebar: readme,
     },
   })
-  .add('CTA', () => {
-    const type = ['local', 'jump', 'external'];
-    // const style = {
-    //   card: 'card',
-    //   text: 'text',
-    //   feature: 'feature',
-    //   buttons: 'button',
-    // };
+  .add('Default', () => {
+    let data, type;
+
+    const copy = ['Lorem ipsum dolor sit amet', 'Consectetur adipisicing elit'];
+    const types = ['local', 'jump', 'external'];
+    const styles = ['text', 'card', 'feature', 'button'];
 
-    // const cta = {
-    //   href: 'https://www.ibm.com',
-    //   text: 'Nunc Pulvinar Nisi',
-    //   target: '_blank',
-    // }
-    const data = {
-      text: {
-        copy: 'Nunc Pulvinar Nisi',
-        href: 'https://www.ibm.com',
-        type: type[1],
-      },
-      card: {
-        href: 'https://www.ibm.com',
-        title: 'Nunc Pulvinar Nisi',
-        type: type[1],
-      },
-      feature: {
-        heading: 'Nunc Pulvinar Nisi',
-        card: {
-          href: 'https://www.ibm.com',
-          title: 'Nunc Pulvinar Nisi',
-          type: type[1],
-          image: {
-            defaultImage: 'https://picsum.photos/id/2/672/672',
-            alt: 'featured link image',
-          },
-        },
-      },
-      button: [
-        {
-          href: 'https://www.ibm.com',
-          copy: 'Nunc Pulvinar Nisi',
-          type: type[1],
-        },
-        {
-          href: 'https://www.example.com',
-          copy: 'Nunc Pulvinar Nisi',
-          type: type[1],
-        },
-      ],
-    };
-    const styles = Object.keys(data);
     const style = select('style', styles, styles[0]);
-    const ctaData = object(`Data (${style})`, JSON.stringify(data[style]));
-    console.log('Button', ctaData);
-    //  const data = object(`Data (${stye})`, date[style])
-    // cta[ctaType] = data;
+    const urlBy = {
+      jump: '#example',
+      local: 'https://ibm.com',
+      external: 'https://example.com',
+    };
+
+    switch (style) {
+      case 'text':
+        type = select('type', types, types[0]);
+        data = {
+          type: type,
+          href: urlBy[type],
+          copy: copy[0],
+        };
+        break;
+      case 'card':
+        type = select('type', types, types[0]);
+        data = {
+          type: type,
+          href: urlBy[type],
+          title: copy[0],
+        };
+        break;
+      case 'feature':
+        type = select('type', types, types[0]);
+        data = {
+          heading: copy[0],
+          card: {
+            type: type,
+            href: urlBy[type],
+            title: copy[1],
+            image: {
+              defaultImage: 'https://picsum.photos/id/672/672',
+              alt: '...',
+            },
+          },
+        };
+        break;
+      case 'button':
+        type = [
+          select('button 1 type', types, types[0]),
+          select('button 2 type', types, types[0]),
```

Comment:
Instead of messing with the `object`, and other data values like the headings and stuff I figured it'd be simpler if we focus on style and type. If they want to play around with the other values they can go to the specific component we're pulling in.